cmd/stolonctl/main.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright (C) 2018 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"os"
+
+	"github.com/gravitational/stolon-app/internal/stolonctl/pkg/defaults"
+
+	"github.com/fatih/color"
+	"github.com/gravitational/trace"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var (
+	kubeConfig      string
```

Comment:
Not necessary at this stage, but it may be worth considering having some defaults that make sense on our systems.